services/src/main/java/org/keycloak/protocol/saml/profile/ecp/authenticator/HttpBasicAuthenticator.java

```diff
@@ -52,17 +53,24 @@ public String getDisplayType() {
 
     @Override
     public String getReferenceCategory() {
-        return null;
+        return "basic";
     }
 
```

Comment:
I'd be down for that, but looking at the error responses, I need to make some updates to this MR.  With the implementation we've got running, we're sending back JSON responses specific to the docker client for display back to the user.  I suppose we'd want to do that as well here, but that's probably going to mean having a docker-specific HTTP Basic authenticator since we'll be crafting custom error responses.  Any other way to do that?